views.py

```diff
@@ -393,13 +393,21 @@ def export_articles_all(request):
     """
     stage = request.GET.get('stage')
 
-    articles = submission_models.Article.objects.filter(
-        journal=request.journal,
-    ).exclude(
-        stage=submission_models.STAGE_UNSUBMITTED,
-    ).select_related(
-        'correspondence_author',
-    )
+    if request.POST:
+        article_id = request.POST.get('article_id')
+    else:
+        article_id = None
+
+    if article_id:
+        articles = submission_models.Article.objects.filter(pk=article_id)
```

Comment:
This should also filter by `request.journal` where available to avoid cross-journal  attacks